### PR TITLE
add formatter.js v0.1.5 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/firstopinion/formatter.js
**Watch 88
  Star 2,312
 Fork 182**
@Amomo  could u help me check this PR for #6556 ? thank you!

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thanks~

File: ajax/libs/formatter.js/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "format",
+    "pattern"
+  ],
+  "homepage": "https://github.com/firstopinion/formatter.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/firstopinion/formatter.js.git",
+    "basePath": "dist",
+    "files": [
+      "*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/formatter.js/package.json]: Why not `"**/*"`?

**Comment by maruilian11** [comment]
@Amomo could you help me check this PR again? thank you!

**Comment by PeterDaveHello** [comment]
Failed.

**Comment by maruilian11** [comment]
@Amomo the commit has passed the test. 
could you help me check this PR again? thank you!

**Comment by Amomo** [comment]
Hi @maruilian11 sorry for late.
I think that you use `"**/*"` in auto-update config, so the files in `"amd/"` and `"common/"` folder should also add in this PR, right?

**Comment by maruilian11** [comment]
@Amomo could you help me check this PR again?

**Pull request merged**